### DISCUSSION TITLE
How do I change the startup shell and user?

### DISCUSSION THREAD

**Discussion by dullbananas** [ish-app/ish#1440] [Q&A] [answered]
By default, iSH starts with `ash` logged in as `root`. How do I change that?

**Comment by dullbananas** [comment, marked as answer]
(Answered from discord)

Hold down the settings button, and it appears after 5 seconds. Scroll to the bottom and you will see the needed options.

**Reply by Ciel-MC** [reply]
Hello, just found this post, but it doesn't seem to work anymore? Is the functionality removed, or maybe it's just not present in the App Store version?

**Reply by electblake** [reply]
it's 10 seconds not 5
https://www.reddit.com/r/ish/comments/pcaglv/user_login_prompt_is_it_possible_in_ish/

**Reply by khurshid-alam** [reply]
Ish crashing after entering user password on ios 16. Any idea why ?
